assets/music/Threadbare_Main_Full_Quiet.ogg.import

```diff
@@ -0,0 +1,19 @@
+[remap]
+
+importer="oggvorbisstr"
+type="AudioStreamOggVorbis"
```

Comment:
What parameters did you use to convert them to Ogg Vorbis?

Reply:
Oh, I just opened them in Audacity and exported as .ogg, I don't know much about this, what do you recommend?

Reply:
That's probably fine, I was just curious what the quality settings were. We can always revise it.

Reply:
The settings I used were the default:
- Sample Rate: 44100hz
- Quality: 5